src/package-vsix.js

```diff
@@ -17,26 +17,26 @@ const repository = {
     "url": "https://github.com/theia-ide/vscode-builtin-extensions"
 };
 
-// bump to publish
-let version = '1.39.1-prel';
+const vscodePck = JSON.parse(fs.readFileSync(vscode('package.json'), 'utf-8'));
+let version = vscodePck.version || '0.0.1';
```

Comment:
when vsix packaging a `latest`, the version of the builtins is taken from vscode's package.json